hibike/README.md

```diff
@@ -223,6 +223,7 @@ Device Type Enumeration:
 |         |                | 12           | enc_pos             | float      | yes   | yes    |
 |         |                | 13           | enc_vel             | float      | yes   | no     |
 |         |                | 14           | motor_current       | float      | yes   | no     |
+|         |                | 15           | deadband            | float      | no    | yes    |
```

Comment:
In general, we left values by default not readable, but yea i agree to make deadband readable. Will make necessary changes for this.